easybuild/easyconfigs/t/TopHat/TopHat-2.0.13-goolf-1.7.20.eb

```diff
@@ -0,0 +1,36 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'ConfigureMake'
+
+name = 'TopHat'
+version = '2.0.13'
+
+homepage = 'http://ccb.jhu.edu/software/tophat/'
+description = """TopHat is a fast splice junction mapper for RNA-Seq reads."""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://ccb.jhu.edu/software/tophat/downloads/']
+
+patches = ['tophat-2.0.13-zlib.patch']
+
+dependencies = [
+    ('Boost', '1.55.0'),
```

Comment:
For the record, `Boost` is the most tricky package on HPC platforms ever,
since it routinely may not be compatible with itself (yes, talking minor
versions here).

So, I wouldn't be surprised a dent, if someone comes forward and says:
"please don't". However, this should be stated explicitly or, assumed as
tricky step...


On 1 June 2015 at 12:35, Kenneth Hoste <notifications@github.com> wrote:

> In easybuild/easyconfigs/t/TopHat/TopHat-2.0.13-goolf-1.7.20.eb
> <https://github.com/hpcugent/easybuild-easyconfigs/pull/1476#discussion_r31417645>
> :
>
> > +name = 'TopHat'
> > +version = '2.0.13'
> > +
> > +homepage = 'http://ccb.jhu.edu/software/tophat/'
> > +description = """TopHat is a fast splice junction mapper for RNA-Seq reads."""
> > +
> > +toolchain = {'name': 'goolf', 'version': '1.7.20'}
> > +toolchainopts = {'optarch': True, 'pic': True}
> > +
> > +sources = [SOURCELOWER_TAR_GZ]
> > +source_urls = ['http://ccb.jhu.edu/software/tophat/downloads/']
> > +
> > +patches = ['tophat-2.0.13-zlib.patch']
> > +
> > +dependencies = [
> > +    ('Boost', '1.55.0'),
>
> Boost 1.58 would make sense, since it has been the latest version for a
> while now?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hpcugent/easybuild-easyconfigs/pull/1476/files#r31417645>
> .
>



-- 


echo "sysadmin know better bash than english"|sed s/min/mins/ \
  | sed 's/better bash/bash better/' # signal detected in a CERN forum